src/Orc.Automation.Tests/NUnit/Commands/ScenarioCommand.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace Orc.Automation.Tests;
+
+using System;
+using Catel;
+using NUnit.Framework.Internal;
+using NUnit.Framework.Internal.Commands;
+using ScenarioManagement;
+
+public class ScenarioCommand : DelegatingTestCommand
+{
+    private readonly AutomationScenario _scenario;
+
+    public ScenarioCommand(TestCommand innerCommand, AutomationScenario scenario)
+        : base(innerCommand)
+    {
+        ArgumentNullException.ThrowIfNull(scenario);
+
+        _scenario = scenario;
+    }
+
+    public override TestResult Execute(TestExecutionContext context)
```

Comment:
I think this should be async by design (or at least have both async and sync implementation)

Reply:
it's override of DelegatingTestCommand (NUnit) - for NUnit internal use